### ISSUE TITLE
SDX62/65 Loaders

### ISSUE THREAD

**Issue by stich86** [bkerler/Loaders#82]
Hi all,

Anyone has SDX62/65 loaders? 

Thx in advance

**Comment by RenateUSB** [comment]
Nope. Please state HWID and Hash.

**Comment by stich86** [comment]
here is it:

```
Qualcomm Sahara / Firehose Client V3.61 (c) B.Kerler 2018-2023.
main - Using loader prog_firehose_lite.elf ...
main - Waiting for the device
main - Device detected 🙂
sahara - Protocol version: 2, Version supported: 1
main - Mode detected: sahara
sahara -
------------------------
HWID:              0x001990e100040000 (MSM_ID:0x001990e1,OEM_ID:0x0004,MODEL_ID:0x0000)
Unknown CPU, please send log as issue to https://github.com/bkerler/edl
PK_HASH:           0xd741a1a1a10b5c6c923fb67be71b1f78ec8823fad892cee268a2edba804ec1f53dddd70e28b4b46da0a585430305bae2000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
Serial:            0xf58ed4c6

sahara - Protocol version: 2, Version supported: 1
sahara - Uploading loader prog_firehose_lite.elf ...
sahara - 32-Bit mode detected.
sahara - Firehose mode detected, uploading...
```

Not sure if it's secured boot enabled device (ZTE MC889)? I've tried a Loader from Quectel/SIMCOM module that are using SDX62 but without success

**Comment by stich86** [comment]
I can confirm that latest 888 revs, 889 and 888 ultra have secure boot enabled and locked bootloader. So without a signed firehose no way to backup or restore

**Comment by RenateUSB** [comment]
I still don't see that hash anywhere.

**Comment by bkerler** [comment]
@stich86 any success ?

**Comment by stich86** [comment]
> @stich86 any success ?

Currently not..
There is a loader but not public available, so that person wants to avoid any legal issue with ZTE

let's hope that someone will leak a public firmware asap :(

**Comment by stich86** [comment]
Anyway I've some x62/65 loaders from Quectel and SIMCOMM.
I'll share them when back from holidays :)

**Comment by bkerler** [comment]
That's a shame. I wanted to dig into sdx62 and zte mf888 sounded like a viable option. Guess it's quectel then again.

**Comment by stich86** [comment]
> That's a shame. I wanted to dig into sdx62 and zte mf888 sounded like a viable option. Guess it's quectel then again.

If it's a plain MC888, Quectel loader of RM520 will work. But if it's 888A or D no, needs signed loader

**Comment by stich86** [comment]
Hi @bkerler 

here are the loaders:

- Quectel RM520N (SDX62)
- SIMCOMM 8260 and 8262 (SDX62)
- SIMCOMM 8380 (SDX65)

[Loader_SDX6X.zip](https://github.com/user-attachments/files/16315729/Loader_SDX6X.zip)

Feel free to add to the collection :)

**Comment by RenateUSB** [comment]
That looks good, but hashes as:
d9357db88795b5a8 afaebfd9ab08a569 cc8e519f6c689723 759f4e6915ca3466 e98b5a3282678bdf 63673d8517bb0c5b
So, it's not valid for the device posted above.
It's using a Qualcomm test key.

**Comment by stich86** [comment]
> That looks good, but hashes as:
> 
> d9357db88795b5a8 afaebfd9ab08a569 cc8e519f6c689723 759f4e6915ca3466 e98b5a3282678bdf 63673d8517bb0c5b
> 
> So, it's not valid for the device posted above.
> 
> It's using a Qualcomm test key.

Yes I know, newer ZTE device are secure booted and needs signed loader 

Currently not public available

**Comment by seanozman** [comment]
I have an inseego m3200 x pro 5g Mifi from Telstra ( m
3000 series, M3100 from T-Mobile )  has the same hw ID, also need a loader to backup the firmware.

PS G:\edl> py -3 .\edl.py
Qualcomm Sahara / Firehose Client V3.52 (c) B.Kerler 2018-2021.
main
main - [LIB]: [33mPlease first install libusb_win32 driver from Zadig[0m
main - Trying with no loader given ...
main - Waiting for the device
main - Device detected :)
main - Mode detected: sahara
Device is in EDL mode .. continuing.
sahara -
------------------------
HWID: 0x001990e1001a0001 (MSM_ID:0x001990e1,OEM_ID:0x001a,MODEL_ID:0x0001)
Unknown CPU, please send log as issue to https://github.com/bkerler/edl
PK_HASH: 0xeb4f4206e731274fca394baa87aac8ccb7b76a9ed5b04801d0f858641030e7c9
Serial: $$$$$$$

**Comment by MrJeff89** [comment]
> Hi [@bkerler](https://github.com/bkerler)
> 
> here are the loaders:
> 
>     * Quectel RM520N (SDX62)
> 
>     * SIMCOMM 8260 and 8262 (SDX62)
> 
>     * SIMCOMM 8380 (SDX65)
> 
> 
> [Loader_SDX6X.zip](https://github.com/user-attachments/files/16315729/Loader_SDX6X.zip)
> 
> Feel free to add to the collection :)

Hello @stich86 ,

This loader can't use.
Please advise.

Thanks
